aesara/tensor/math_opt.py

```diff
@@ -3539,3 +3621,27 @@ def local_reciprocal_1_plus_exp(fgraph, node):
 )
 register_canonicalize(local_sigmoid_logit)
 register_specialize(local_sigmoid_logit)
+
+
+fastmath = EquilibriumDB()
+
+optdb.register("fastmath", fastmath, 1, "fast_run", "mul")
+
+fastmath.register(
+    "dist_over_add_opt",
+    in2out(distribute_over_add_opt, ignore_newtrees=True),
+    1,
```

Comment:
`EquillibriumDB` doesn't take a position argument like `SequenceDB`; plus, `SequenceDB` was recently change so that the position argument is now a keyword.  

This numeric value is probably being interpreted as a tag, and it should really raise an exception (looks like we need an issue for that).